### PR TITLE
Fix the file path for production deployment

### PR DESCRIPTION
**Pull request by mscherer**
@sgoggins , I deployed as we discussed last week, but it seems to fail due to path issues.

### CODE REVIEW

File: augurface/vue.config.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
 // #SPDX-License-Identifier: MIT
 module.exports = {
```
Comment (sgoggins) [inline augurface/vue.config.js:2]: @mscherer : This change makes me wonder if you are running `augurface` with the intention of having it be Augur's primary frontend, or if you are using it to load repos (which is what it is for, and its not quite right/perfect yet.).  The main frontend is in the `frontend` directory. 

There is also another project, https://github.com/chaoss/augur_view that we are developing new frontend "stuff" in.

**Review by sgoggins — COMMENTED** [review]
@mscherer : Just a question to clarify intentions.

**Comment by sgoggins** [comment]
@mscherer : tracking identities .. you are the person who is installing for San Diego that I zoomed with? Sorry. :)

**Comment by mscherer** [comment]
@sgoggins yep, that's me. Forgot my profile was private, so you can't see my orgs nor my name :/

And yes, I finally deployed the proxy we talked about, but index.html point to server.tld/augurface/foo.js, while we serve them directly from the root directory. I can work around using configuration magic, but figured this would be easier.

But if that's supposed to be consumed by more than just our instance, I guess I need to propose another fix.

**Comment by sgoggins** [comment]
@mscherer : It would be better for Augur if we kept the `/augurface` path in the URL, simply because there's another interface served up at `/` .. Is that possible.

**Comment by sgoggins** [comment]
@mscherer : I will merge this and we can see what happens.

**Pull request merged**